framework/web/js/source/jquery.ba-bbq.js

```diff
@@ -480,6 +482,10 @@
         // into its component parts.
         keys = key.split( '][' ),
         keys_last = keys.length - 1;
+
+      if (prohibitedKeys.includes(key)) {
+        return;
+      }
```

Comment:
Indentation not correct.

Reply:
I have fixed the formatting. The indentation still looks a bit weird but it is correct if you look at all of the context above.